### DISCUSSION TITLE
Allow world in weather command

### DISCUSSION THREAD

**Discussion by Josh65-2201** [EssentialsX/Essentials#5174] [Ideas and Feature Suggestions] [answered]
Weather only effects the current world even when putting a world name after selecting the type and length of weather. Would be nice to be able to do `/weather sun 100 lobby`

**Comment by Warriorrrr** [comment, marked as answer]
You can specify the world by putting the world name as the first arg, not the last. i.e. /weather lobby sun 100

**Reply by Josh65-2201** [reply]
Doesn't work just says Error for input string 'sun'
